server/core/src/access_time_queue.cpp

```diff
@@ -34,6 +37,14 @@ namespace irods::access_time_queue
             return;
         }
 
+        if (_queue_name_prefix.empty()) {
+            THROW(CONFIGURATION_ERROR, fmt::format("{}: Access time queue name is empty.", __func__));
```

Comment:
not sure if this is what you meant...

```suggestion
            THROW(CONFIGURATION_ERROR, fmt::format("{}: Access time queue name prefix is empty.", __func__));
```

Reply:
Fixed.